docs/0.4/creating_splinter_circuits.md

```diff
@@ -211,54 +231,55 @@ proposal.
    proposal and for interacting with the circuit once it is approved. For
    example:
 
-   ```
-   root@splinterd-beta:/# splinter circuit proposals --url http://splinterd-beta:8085
-   ID            MANAGEMENT MEMBERS                      COMMENTS
-   01234-ABCDE   grid       alpha-node-000;beta-node-000
-   ```
+    ```
+    root@splinterd-beta:/# splinter circuit proposals
+    ID          NAME MANAGEMENT MEMBERS                      COMMENTS PROPOSAL_TYPE
+    01234-ABCDE -    grid       alpha-node-000;beta-node-000 -        Create
+    ```
 
 1. As on the first node, save the ID in the `CIRCUIT_ID` environment variable,
    which simplifies entering `splinter` commands.
 
-   ```
-   root@splinterd-beta:/# export CIRCUIT_ID=01234-ABCDE
-   ```
+    ```
```

Comment:
Not a big deal, but are these spaces intentional?

Reply:
Nah, original documentation had funky left spacing, 1 tab + 1 space, so my inserting a new version of that and attempting to get it spaced correctly was simpler to double tab rather than tab + space to match the original indentation